src/model/docker.ts

```diff
@@ -72,11 +73,15 @@ const Docker = {
         --volume "${githubHome}":"/root:z" \
         --volume "${githubWorkflow}":"/github/workflow:z" \
         --volume "${workspace}":"/github/workspace:z" \
+        --volume "${actionFolder}/steps":"/steps:z" \
+        --volume "${actionFolder}/entrypoint.sh":"/entrypoint.sh:z" \
         ${sshAgent ? `--volume ${sshAgent}:/ssh-agent` : ''} \
         ${sshAgent ? '--volume /home/runner/.ssh/known_hosts:/root/.ssh/known_hosts:ro' : ''} \
         ${useHostNetwork ? '--net=host' : ''} \
         ${githubToken ? '--env USE_EXIT_CODE=false' : '--env USE_EXIT_CODE=true'} \
-        ${image}`;
+        ${image} \
+        -- \
+        /usr/bin/bash /entrypoint.sh` ;
```

Comment:
[](https://user-images.githubusercontent.com/20756439/160925541-594be8b0-b8f4-48ac-9903-855209de796b.png)I believe you don't need to pass the -- in this case, that's only necessary in k8s when passing something to the API and then passing an argument to the resulting resource.

In this case we're running a command in the docker image directly. So we can do the command part without the double hyphens

![image](https://user-images.githubusercontent.com/20756439/160927131-521c73e9-cfaf-43c0-abea-d40c9e9af40b.png)

```suggestion
        ${image} \
        /usr/bin/bash /entrypoint.sh` ;
```